docs/conf/scalar-loki-stack-custom-values.yaml

```
@@ -0,0 +1,55 @@
+promtail:
+  config:
+    snippets:
+      # -- `scapeConfigs` is exactly the part of https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs
+      # -- The value will be created as a Kubernetes ConfigMap and then mounted to the Promtail POD.
+      # -- Not really need to change this value. It's set to scrape all logs of Scalar DL/DB PODs by using regular expression.
+      scrapeConfigs: |
+        # -- the `scalardl` job scrapes the logs from all PODs of Scalar Ledger, Scalar Auditor and related Envoy
```

```suggestion
        # -- the `scalardl` job scrapes all the logs from Scalar Ledger Pods, Scalar Auditor Pods, and the corresponding Envoy Pods
```

Also, it's a bit picky, but it's not P-O-D, and it's /pɑːd/.